modular_ss220/bureaucracy/code/photocopier.dm

```diff
@@ -0,0 +1,113 @@
+/obj/machinery/photocopier
+	/// Selected form's category
+	var/category = ""
+	/// Selected form's id
+	var/form_id = ""
+	/// List of available forms
+	var/list/forms
+	/// Selected form's datum
+	var/datum/bureaucratic_form/form
+	/// Printing sound
+	var/print_sound = 'sound/goonstation/machines/printer_dotmatrix.ogg'
+
+/obj/machinery/photocopier/Initialize(mapload)
+	. = ..()
+	forms = new
+
+/obj/machinery/photocopier/attack_ai(mob/user)
+	src.add_hiddenprint(user)
```

Comment:
```src``` is necessary here?

Reply:
No. The unnoticed consequence of copy-paste.